src/Utils/OrientationUtils.js

```diff
@@ -233,15 +337,20 @@ export default {
      * @param {string|Projection} crsOrProj - the CRS of the target frame or its
      * proj4-compatible object.
      *
-     * @return {OrientationFunction} the orientation function
+     * @param {Coordinates} [coordinates]  coordinates the origin of the local East North Up
+     * (ENU) frame
+     * @param {THREE.Quaternion} [target=new THREE.Quaternion()] output Quaternion
+     * @return {FunctionOrQuaternion} The target quaternion if coordinates is defined, otherwise, a function to compute it from coordinates.
      */
-    quaternionToCRS(crsOrProj) {
+    quaternionToCRS(crsOrProj, coordinates, target = new THREE.Quaternion()) {
```

Comment:
May be replace by  `quaternionToCoordinates` and so, and use Coordinates.crs to found `crsOrProj`.

Reply:
I do not like ` quaternionToCoordinates ` as a function name, because it computes the quaternion modeling the rotation from a Crs **to a CRS**, rather than to the coordinates of a 3D point (which is only the origin of the local frame).

Reply:
ok, my bad, I will come back to the reviews because I didn't understand the role of the functions